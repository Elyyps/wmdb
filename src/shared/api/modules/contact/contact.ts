export interface IContact {
  buttonText: string;
  category: string;
  content: string;
  formContent: string;
  formTitle: string;
  personsNumber: string;
  personsNumberIcon: string;
  title: string;
}
