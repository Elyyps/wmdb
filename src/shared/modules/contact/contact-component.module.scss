.other-activities {
  margin-top: 40px;
}
.contact-telephone-info {
  font-size: 14px;
}

.contact-telephone-number {
  flex-direction: column;
  margin-top: 8px;

  margin-left: 24px;
  span {
    font-weight: bold;
    padding-left: 8px;
  }
  p {
    font-weight: lighter;
  }
}
.contact {
  &-content {
    margin: 20px 0 20px 0;
  }
  &-grid {
    display: flex;
    @media (max-width: $breakpoint-medium-max) {
      flex-wrap: wrap;
    }
  }
  &-back {
    display: flex;
    align-items: center;
    font-size: 14px;
    color: $global-link-color;
    padding: 16px 0;
    > div {
      margin-right: 8px;
    }
    * {
      stroke: $global-link-color;
    }
    &:hover {
      text-decoration: none;
      * {
        stroke: $global-link-hover-color;
      }
    }
  }
  &-modal {
    display: block;
    width: 100%;
    @media (min-width: $breakpoint-medium) {
      margin: 24px 0;
    }
    @media (max-width: $breakpoint-small-max) {
      position: fixed;
      left: 0;
      right: 0;
      top: 0;
      bottom: 0;
      background-color: $global-inverse-color;
      z-index: 1;
      padding: 16px;
      box-sizing: border-box;
      overflow: scroll;
    }
    &-close {
      cursor: pointer;
      position: absolute;
      left: 16px;
      top: 16px;
      background-color: transparent;
      border: none;
      outline: none;
    }
    &-head {
      justify-content: center;
      padding-bottom: 16px;
      border-bottom: 1px solid $global-border;
      margin: 0 -16px;
      display: none;
      @media (max-width: $breakpoint-small-max) {
        display: flex;
      }
      h5 {
        margin: 0;
      }
    }
  }
  &-form {
    &-label {
      background-color: #e8f2fc;
      padding: 16px;
      margin: 0 -16px 16px -16px;
      display: none;
      @media (max-width: $breakpoint-small-max) {
        display: block;
      }
      ul {
        margin: 0;
        display: flex;
        justify-content: center;
      }
    }
    &-radio-label {
      margin-bottom: 8px;
      font-weight: 600;
    }
    &-bottom {
      margin-top: 16px;
      button {
        margin-top: 24px;
      }
    }
    &-head {
      display: flex;
      margin: 0 -16px 0 -16px;
      > * {
        padding: 0 16px;
      }
    }
  }
  &-data {
    font-size: 14px;
    padding: 16px 0;
    display: flex;
    flex-direction: column;
    p {
      margin-bottom: 16px;
    }
    &-bottom {
      margin-top: 28px;
      display: flex;
      justify-content: center;
    }
    &-info {
      display: flex;
      padding: 16px;
      flex-direction: column;
      border: solid 1px #e69c27;
      color: #e69c27;
      background-color: #fff8ed;
      border-radius: $global-border-radius;
      margin: 12px 0 24px 0;
      align-items: center;
      line-height: 1.57;
      font-size: 14px;
      a {
        color: #e69c27;
        text-decoration: none;
      }
    }
  }
  &-main {
    flex-basis: 640px;
    flex-shrink: 0;
    position: relative;
    &:after {
      display: none;
    }
    @media (max-width: $breakpoint-small-max) {
      &:after {
        display: block;
        content: "";
        position: absolute;
        left: 0;
        right: 0;
        top: 0;
        bottom: 0;
        background-image: linear-gradient(to top, #ffffff, rgba(255, 255, 255, 0));
      }
    }
    @media (max-width: $breakpoint-large-max) {
      flex-basis: 50%;
    }
    @media (max-width: $breakpoint-medium-max) {
      flex-basis: 100%;
    }
    @media (max-width: $breakpoint-small-max) {
      height: 520px;
      overflow: hidden;
      &.fullheight {
        height: 100%;
        &:after {
          display: none;
        }
      }
    }
    &-head {
      margin-bottom: 12px;
    }
    h1,
    h2,
    h3,
    h4,
    h5 {
      margin-bottom: 8px;
      + p {
        margin-top: 0;
      }
    }
  }
  &-aside {
    flex-grow: 1;
    padding-left: 48px;
    @media (max-width: $breakpoint-large-max) {
      padding-left: 24px;
    }
    @media (max-width: $breakpoint-medium-max) {
      padding-left: 0;
      margin-top: 24px;
    }
    h2 {
      margin-bottom: 12px;
      color: #02906f;
    }
  }
  &-grid {
    position: relative;
    z-index: 1;
  }
  &-body {
    padding: 42px 0;
    position: relative;
    background-size: 5%;
    @media (max-width: $breakpoint-small-max) {
      padding: 24px 0 0;
    }
    &:after {
      content: "";
      background: linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0.7));
      position: absolute;
      left: 0;
      right: 0;
      top: 0;
      bottom: 0;
    }
  }
  &-actions {
    display: flex;
    flex-wrap: wrap;
    margin-bottom: 24px;

    &-mail {
      width: 100%;
      margin: 12px 0 12px 0;
      display: flex;
      justify-content: center;
      opacity: 0.6;
      color: #101010;
      &-span {
        margin-left: 7px;
        font-size: 16px;
        font-weight: 600;
        font-style: normal;
        font-stretch: normal;
        line-height: normal;
        letter-spacing: normal;
        text-align: center;
      }
      svg {
        margin: auto;
        width: 100%;
        height: 100%;
      }
      a {
        align-items: center;
        justify-content: center;

        width: fit-content;
      }
    }
  }
  &-list {
    margin: 0;
    padding: 0;
    list-style: none;
    display: flex;
    align-items: center;
    color: $gray;
    font-size: 14px;
    margin: 0 -18px;
    li {
      padding: 0 18px;
      position: relative;
      line-height: 1;
      display: flex;
      > * {
        color: $gray;
        margin-right: 10px;
        svg {
          fill: $gray;
        }
      }
      &:not(:first-child) {
        &:after {
          content: "";
          position: absolute;
          width: 3px;
          height: 3px;
          background-color: $gray;
          display: block;
          left: 0;
          top: 50%;
          transform: translateY(-50%);
          margin-right: 2px;
        }
      }
    }
  }
  &-mobile-actions {
    background: linear-gradient(to top, #ffffff, rgba(255, 255, 255, 0.6));
    display: none;
    width: calc(100% + 32px);
    margin: 0 -16px;
    &-contact {
      display: flex;
      flex-direction: column;
      padding: 0 16px;
    }
    @media (max-width: $breakpoint-small-max) {
      display: flex;
      flex-direction: column;
      justify-content: space-between;
    }
  }
  &-mobile {
    &-bottom {
      background-color: $global-inverse-color;
      display: flex;
      padding: 16px 16px 16px;
      border-top: 1px solid $global-border;
      justify-content: center;
    }
    &-item {
      width: 100%;
      padding: 0 4px;
      > button {
        padding-left: 0;
        padding-right: 0;
      }
    }
    &-head {
      display: flex;
      justify-content: center;
      padding: 0 16px;
      background-color: $global-inverse-color;
      > button {
        background-color: transparent;
        border: none;
        outline: transparent;
        &:hover {
          background-color: transparent;
        }
      }
    }
  }
  &-telephone {
    margin-top: 50px;
    h2 {
      margin-bottom: 24px;
    }
    p {
      margin-bottom: 16px;
    }
    a {
      display: flex;
      align-items: center;
      text-decoration: underline;
      width: fit-content;
      span {
        margin-right: 8px;
      }
    }
    @media (max-width: $breakpoint-small-max) {
      margin: 32px 0 24px 0;
      h2 {
        margin-bottom: 8px;
      }
      p {
        margin: 0;
      }
    }
    &-bottom {
      @media (max-width: $breakpoint-small-max) {
        margin-top: 16px;
      }
    }
  }
}
