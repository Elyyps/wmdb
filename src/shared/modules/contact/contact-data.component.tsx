import React from "react";
import styles from "./contact-component.module.scss";
import { Button } from "@app/core/button";
import LOCATION from "@assets/icons/location.svg";
import { Link } from "react-router-dom";
import { ILink } from "@app/api/core/link";

export interface IContactDataComponentProps {
  contactName: string;
  content: string;
  onClick: () => void;
  phone: ILink;
  phoneButton: ILink;
  priceButton: ILink;
}

const ContactDataComponent = (props: IContactDataComponentProps) => (
  <div className={styles["contact-data"]}>
    <p>{props.content}</p>
    <Button title={props.phoneButton.title} href={props.phoneButton.url} icon={LOCATION} position={"left"} />
    <div className={styles["contact-data-info"]}>
      <span>
        <Link to={props.phone.url}>{props.phone.title}</Link>
      </span>
      <span>{props.contactName}</span>
    </div>
    <Button title={props.phoneButton.title} href={props.phoneButton.url} icon={LOCATION} position={"left"} />
    <div className={styles["contact-data-bottom"]}>
      <Button
        title={props.priceButton.title}
        icon={LOCATION}
        position={"left"}
        variant={"primary"}
        onClick={props.onClick}
      />
    </div>
  </div>
);

export { ContactDataComponent };
