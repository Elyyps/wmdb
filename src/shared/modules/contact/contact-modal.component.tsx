import React from "react";
import styles from "./contact-component.module.scss";
import CLOSE from "@assets/icons/cross.svg";
import { IconComponent } from "@app/core/icon";

export interface IContactModalComponentProps {
  children: any;
  onClick: any;
  title: string;
}

const ContactModalComponent = (props: IContactModalComponentProps) => (
  <div className={styles["contact-modal"]}>
    <div className={styles["contact-modal-head"]}>
      <button onClick={props.onClick} className={styles["contact-modal-close"]}>
        <IconComponent icon={CLOSE} size={"16px"} />
      </button>
      <h5>{props.title}</h5>
    </div>
    {props.children}
  </div>
);

export { ContactModalComponent };
