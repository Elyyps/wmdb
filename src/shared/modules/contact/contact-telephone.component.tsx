import * as React from "react";
import styles from "./contact-component.module.scss";
import { Link } from "react-router-dom";
import Phone from "@assets/icons/phone.svg";
import { ILink } from "@app/api/core/link";
import { IconComponent } from "@app/core/icon";

export interface IContactTelephoneComponentProps {
  button: ILink;
  content: string;
  title: string;
}

const ContactTelephoneComponent = (props: IContactTelephoneComponentProps) => (
  <div className={styles["contact-telephone"]}>
    <h2>{props.title}</h2>
    <p>{props.content}</p>
    <div className={styles["contact-telephone-bottom"]}>
      <Link to={props.button.url}>
        <IconComponent icon={Phone} size={"16px"} fillColor={"#34aadf"} />
        <span>{props.button.title}</span>
      </Link>
    </div>
  </div>
);

export { ContactTelephoneComponent };
