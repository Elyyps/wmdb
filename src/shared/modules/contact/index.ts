export * from "./contact.component";
export * from "./contact-modal.component";
export * from "./contact-telephone.component";
