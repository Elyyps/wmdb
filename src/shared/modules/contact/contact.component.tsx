import * as React from "react";
import styles from "./contact-component.module.scss";
import classNames from "classnames";
import { IconComponent } from "@app/core/icon";
import PATTERN from "@assets/pattern/pattern-wmdb.png";
import Arrow from "@assets/icons/chevron-right.svg";
import { BodyTextComponent } from "@app/core/bodytext";
import { ContactFormComponent } from "@app/core/contact-form";
import { OtherActivitiesComponent } from "../other-activities";
import { generateOtherActivitiesData } from "@app/api/modules/other-activities/end-points";
import { ContactModalComponent } from "./contact-modal.component";
import { ContactTelephoneComponent } from "./contact-telephone.component";
import { ContactDataComponent } from "./contact-data.component";
import { Button } from "@app/core/button";
import { IContact } from "@app/api/modules/contact/contact";
import { ActivitiesComponent } from "../activities";
import { generateActivitiesData } from "@app/api/modules/activities/end-point";

export interface IContactComponentProps {
  contactModule: IContact;
}

const ContactComponent = ({ contactModule }: IContactComponentProps) => {
  const [isActive, setIsActive] = React.useState(false);
  const [isData, setIsData] = React.useState(false);
  const [fullHeight, setFullHeight] = React.useState(false);
  const toggleForm = () => {
    setIsData(false);
    setIsActive(!isActive);
  };
  const toggleData = () => {
    setIsData(!isData);
  };
  const toggleMainContent = () => {
    setFullHeight(!fullHeight);
  };
  const myRef = React.useRef(null);
  const scrollToRef = (ref: any) => window.scrollTo({ top: ref.current.offsetTop, behavior: "smooth" });
  const executeScroll = () => scrollToRef(myRef);

  return (
    <div className={styles["contact-body"]} style={{ backgroundImage: `url(${PATTERN})` }}>
      <div className="uk-container">
        <div className={styles["contact-grid"]}>
          <div className={classNames(styles["contact-main"], { [styles["fullheight"]]: fullHeight })}>
            <div className={styles["contact-title"]}>
              <h2>{contactModule.title}</h2>
            </div>
            <ul className={styles["contact-list"]}>
              <li>{contactModule.category}</li>
              <li>
                <IconComponent icon={contactModule.personsNumberIcon} size={"14px"} />
                {contactModule.personsNumber}
              </li>
            </ul>
            <div className={styles["contact-content"]}>
              <BodyTextComponent html={contactModule.content} style={{ color: "black" }} />
            </div>
            <Button title={contactModule.buttonText} variant={"primary"} fullWidth large onClick={executeScroll} />
            <div className={styles["other-activities"]}>
              <OtherActivitiesComponent otherActivitiesModule={generateOtherActivitiesData()} />
            </div>
            <div className={styles["contact-bottom"]}>
              <ContactTelephoneComponent
                title={"Telefonisch contact"}
                content={"Ook al zijn wij vaak op pad met een groep, wij doen altijd ons best u te woord te staan."}
                button={{ title: "Toon telefoonnummer", url: "" }}
                phone={"+31 020 6154 183"}
                contactName="Roel van den Berg"
              />
            </div>
          </div>
          <div className={styles["contact-aside"]}>
            <div className="uk-visible@m">
              <h2>{contactModule.formTitle}</h2>
            </div>
            <div className="uk-visible@m" ref={myRef}>
              <ContactFormComponent onSubmit={() => alert("submitted")} />
            </div>

            {isActive && (
              <ContactModalComponent title={"Informatie aanvraag"} onClick={toggleForm}>
                <ContactFormComponent onSubmit={() => alert("submitted")} />
              </ContactModalComponent>
            )}
            <br />
            <div className="uk-visible@l">
              <ActivitiesComponent activitiesModule={generateActivitiesData()} />
            </div>
          </div>
          <div className={styles["contact-mobile-actions"]}>
            <div className={styles["contact-mobile-head"]}>
              <Button
                icon={Arrow}
                title={fullHeight ? " Lees meer " : "Read more"}
                position={"left"}
                onClick={toggleMainContent}
              />
            </div>
            {!fullHeight && (
              <div className={styles["contact-mobile-actions-contact"]}>
                <ContactTelephoneComponent
                  title={"Telefonisch contact"}
                  content={"Ook al zijn wij vaak op pad met een groep, wij doen altijd ons best u te woord te staan."}
                  button={{ title: "Toon telefoonnummer", url: "#" }}
                  phone={"+31 020 6154 183"}
                  contactName="Roel van den Berg"
                />
              </div>
            )}
            <div className={styles["contact-mobile-bottom"]}>
              <div className={styles["contact-mobile-item"]}>
                <Button title={"Aanvragen"} variant={"primary"} fullWidth onClick={toggleForm} />
              </div>
            </div>
          </div>
        </div>
      </div>
      {isData && (
        <ContactModalComponent title={"Gegevens"} onClick={toggleData}>
          <ContactDataComponent
            phoneButton={{ title: "Toon telefoonnummer", url: "" }}
            content={
              "Wij zouden het op prijs stellen dat wanneer u via ons contact zoekt of belt, u WegmetdeBaas vermeldt!"
            }
            phone={{ title: "024 1234 5678", url: "tel:024 1234 5678" }}
            contactName={"Marie Autenbach"}
            priceButton={{ title: "Prijsindicatie aanvragen", url: "" }}
            onClick={toggleForm}
          />
        </ContactModalComponent>
      )}
    </div>
  );
};

export { ContactComponent };
