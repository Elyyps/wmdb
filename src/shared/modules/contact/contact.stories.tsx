
import * as React from "react";

import { storiesOf } from "@storybook/react";
import { withA11y } from "@storybook/addon-a11y";
import { withProvider } from "@app/util";
import { ContactComponent } from "./contact.component";

storiesOf("Contact", module)
  .addDecorator(withA11y)
  .addDecorator(withProvider)
  .add("Basic implementation", () => (
    <ContactComponent />
  ));

  