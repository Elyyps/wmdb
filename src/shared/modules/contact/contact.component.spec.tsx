import React from "react";
import { shallow } from "enzyme";
import { ContactComponent } from "./contact.component";

describe("[Contact]", () => {
  it("should render component without crashing", () => {
    const renderedComponent = shallow(<ContactComponent />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
