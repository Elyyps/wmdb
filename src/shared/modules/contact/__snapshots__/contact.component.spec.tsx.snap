// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Contact] should render component without crashing 1`] = `
<div
  className="contact-body"
  style={
    Object {
      "backgroundImage": "url(pattern-wmdb.png)",
    }
  }
>
  <div
    className="uk-container"
  >
    <div
      className="contact-grid"
    >
      <div
        className="contact-main"
      >
        <div
          className="contact-title"
        >
          <h2>
            Wie is de mol?
          </h2>
        </div>
        <ul
          className="contact-list"
        >
          <li>
            BE Eventgroup
          </li>
          <li>
            <IconComponent
              icon="rooms.svg"
              size="14px"
            />
            10 - 1000 personen
          </li>
        </ul>
        <div
          className="contact-content"
        >
          <BodyTextComponent
            html="  <p>
Maak kennis met het spel Wie is de Mol? Leuk, leukst en nog veel leuker. Het spel boordevol met verrassingen, wie
o wie bedriegt het gezelschap?
</p>
<p>
Na het welkomstwoord volgt er een uitleg over het spel. De deelnemers van het bedrijfsuitje / teamuitje worden in
subgroepen verdeeld. De mol zit zonder dat de gasten het weten al in elke groep. Anoniem vanuit een ander team in
uw groep geplaatst met als doel saboteren!! De Mol zal de hele dag meeloeren over uw schouders terwijl de ludieke
opdrachten uitgevoerd worden. Wie vertrouwd u in uw team om de opdrachten namens uw team uit te laten voeren?
</p>
<p>
Na de uitleg lopen de deelnemers een route door het centrum van Best. Tijdens deze tocht proberen de groepen geld
te verdienen, door goed uitgevoerde opdrachten. Zonder Creativiteit en samenwerking zal men de opdrachten niet
kunnen vervullen. In elke groep zit een MOL. Deze behoort eigenlijk bij een andere sub groep van uw gezelschap. De
MOL probeert opdrachten dwars te bomen. Hoe meer hij hier in slaagt hoe meer geld hij verdient voor zijn eigen
groep.
</p>
<h5>Kinderfeestje</h5>
<p>
Als iedereen is voorzien van een natje vervolgen we de tocht van Wie is de MOL. Na deze pauze zijn er al
deelnemers die een vermoeden hebben van wie de MOL nu eigenlijk is. Deze gasten \\"proberen\\" dit nog even voor zich
te houden. Het rest jullie nu nog enkele opdrachten te doen voordat de antwoorden bekend worden gemaakt.
</p>
<p>
Uiteraard zal de prijsuitreiking voor verrassingen zorgen. De mollen worden ontmaskerd en de reacties komen los!!
Niet gedacht, wel gedacht etc. etc. Tijdens de borrel krijgen de Mollen de kans om hun 'teamgenoten' te vertellen
hoe zij de gehele dag zijn misleid. Het perfecte bedrijfsuitje om uw collega's beter te leren kennen. Een van de
vele mooie momenten van deze dag.
</p>

<h5>Ligging</h5>
<p>
Op ongeveer de helft van de tocht zal de Mol jullie naar een horecagelegenheid proberen te loodsen. De meeste
deelnemers zijn op dit moment Mister X al tegen gekomen.
</p>

"
          />
        </div>
        <Button
          fullWidth={true}
          large={true}
          onClick={[Function]}
          title="Ontvang informatie / offerte"
          variant="primary"
        />
        <div
          className="other-activities"
        >
          <OtherActivitiesComponent
            otherActivitiesModule={
              Object {
                "buttonTitleClosed": "Bekijk 6 meer activiteiten",
                "buttonTitleOpened": "Bekijk 6 minder activiteiten",
                "cards": Array [
                  Object {
                    "buttonText": "",
                    "image": "oasis.jpg",
                    "title": "U Deugt Wij Deugen",
                  },
                  Object {
                    "buttonText": "",
                    "image": "img02.png",
                    "title": "The Ultimate Expedition",
                  },
                  Object {
                    "buttonText": "",
                    "image": "rectangle.png",
                    "title": "Het Mislukte Feest",
                  },
                  Object {
                    "buttonText": "",
                    "image": "img-big.jpg",
                    "title": "Hou van Holland Stadsspel",
                  },
                  Object {
                    "buttonText": "",
                    "image": "rectangle.png",
                    "title": "Het Mislukte Feest",
                  },
                  Object {
                    "buttonText": "",
                    "image": "img-big.jpg",
                    "title": "Hou van Holland Stadsspel",
                  },
                  Object {
                    "buttonText": "",
                    "image": "rectangle.png",
                    "title": "Het Mislukte Feest",
                  },
                  Object {
                    "buttonText": "",
                    "image": "img-big.jpg",
                    "title": "Hou van Holland Stadsspel",
                  },
                  Object {
                    "buttonText": "",
                    "image": "rectangle.png",
                    "title": "Het Mislukte Feest",
                  },
                  Object {
                    "buttonText": "",
                    "image": "img-big.jpg",
                    "title": "Hou van Holland Stadsspel",
                  },
                ],
                "numberItemsShowing": 4,
                "title": "Andere activiteiten van dit bedrijf",
              }
            }
          />
        </div>
        <div
          className="contact-bottom"
        >
          <ContactTelephoneComponent
            button={
              Object {
                "title": "Toon telefoonnummer",
                "url": "",
              }
            }
            content="Ook al zijn wij vaak op pad met een groep, wij doen altijd ons best u te woord te staan."
            title="Telefonisch contact"
          />
        </div>
      </div>
      <div
        className="contact-aside"
      >
        <div
          className="uk-visible@m"
        >
          <h2>
            Vrijblijvende offerte / prijsindicatie
          </h2>
          <p>
            Vraag vrijblijvend een prijsindicatie aan en ontvang informatie op maat!
          </p>
          <div
            className="contact-actions"
          >
            <Button
              fullWidth={true}
              large={true}
              onClick={[Function]}
              title="Ontvang informatie / offerte"
              type="button"
              variant="primary"
            />
            <div
              className="contact-actions-mail"
            >
              <LinkComponent
                to=""
              >
                <IconComponent
                  icon="mail.svg"
                  size="15px"
                />
                <span
                  className="contact-actions-mail-span"
                >
                  Deel via mail
                </span>
              </LinkComponent>
            </div>
          </div>
        </div>
        <div
          className="uk-visible@m"
        >
          <AdBannerComponent
            advertisement="Advertentie"
            backgroundImage="rectangle.png"
            button={
              Object {
                "title": "Meer info",
                "url": "/",
              }
            }
            content="Surfen? Dat doe je in Zeeland."
          />
        </div>
      </div>
      <div
        className="contact-mobile-actions"
      >
        <div
          className="contact-mobile-head"
        >
          <Button
            icon="chevron-right.svg"
            onClick={[Function]}
            position="left"
            title="Read more"
          />
        </div>
        <div
          className="contact-mobile-actions-contact"
        >
          <ContactTelephoneComponent
            button={
              Object {
                "title": "Toon telefoonnummer",
                "url": "",
              }
            }
            content="Ook al zijn wij vaak op pad met een groep, wij doen altijd ons best u te woord te staan."
            title="Telefonisch contact"
          />
        </div>
        <div
          className="contact-mobile-bottom"
        >
          <div
            className="contact-mobile-item"
          >
            <Button
              fullWidth={true}
              onClick={[Function]}
              title="Aanvragen"
              variant="primary"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
